Automattic/wp-calypso
# Fix Help Center Icon Alignment

This centers the HelpIcon in the center of the button.

### Testing in Calypso
1. Visit the live link below.
2. The Icon should be smack center in the Master bar.

### Testing in the editor
1. cd into `apps/editing-toolkit`.
2. run `yarn dev --sync`.
3. sandbox a simple site.
4. Visit that sites editor.
5. Open the Help Center.
6. Icon should be smack center. 

### Testing in wp-admin
wp-admin uses its own instance of the icon. It's unaffected by this change.

Context here: p9Jlb4-6wN-p2#comment-7047
